templates/taxi/driver_list.html

```diff
@@ -5,10 +5,16 @@
   {% if driver_list %}
     <ul>
       {% for driver in driver_list%}
-        <li>
+        {% if driver.username == user.username %}
+          <li>
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
The link to the driver's detail page is correctly implemented using the `driver-detail` URL pattern. Ensure that the `driver-detail` view is correctly set up to handle this URL.